Sources/Sentry/include/SentryProfiler.h

```diff
@@ -44,7 +44,7 @@ SENTRY_EXTERN_C_END
 /**
  * Start a profiler, if one isn't already running.
  */
-+ (void)startWithTracer:(SentryId *)traceId;
++ (BOOL)startWithTracer:(SentryId *)traceId;
```

Comment:
`m`: I think we should rename `traceID` to `concurrencyID` to state it's not the traceID. I would also rename all occurrences in `traceID` in `SentryProfiledTracerConcurrency`.

Reply:
i'll call it `internalID` to match the declaration's name in SentryTracer